### ISSUE TITLE
Arch Linux

### ISSUE THREAD

**Issue by tyumici** [tyumici/the-plan#1]
The  change to.

**Comment by tyumici** [comment]
- [ ] Files moved to NAS
- [ ] Possibly iffy drive removed
- [ ] Compat check on used programs
- [ ] Bookmarks backed up
- [ ] Bookmarks organized

**Comment by tyumici** [comment]
Trying to format that drive over a dock is so slow omg. I'm giving up that.

**Comment by tyumici** [comment]
Oh man what a fucking journey. I've been on Arch for about a full week now. It's been mostly fine, and everything is setup just fine.

I did try to get Clip Studio running off a Windows VM, but it was not behaving well. I've elected to just use Krita, and so far it seems very good.

All drivers were easy enough to find and install for printers, controllers, and my drawing tablet. 

Managed to resize my initial LVM for root without breaking my install so that was exciting,

Current thing that has me wanted to re-install is that Discord for SOME REASON is changing ALL my audio profiles when joining a voice channel. I have not seen any posts with this issue and it is driving me mad. It is resolvable, but still has to be rectified anytime I join a channel.

All other audio things have actually been good otherwise.

Worst thing was an immediate issue with screen resolution. No idea why but Wayland on KDE would not detect my monitor. X11 works though so I'm set to stay on that.

Another issue was Steam, for some reason my two profiles can't share the same games drive unless it's the home drive. Kinda annoying but I can work around it.

**Comment by tyumici** [comment]
I can count this done, but want to give it a month to have a final say.

@tyumici close this on September 30

**Comment by tyumici** [comment]
Updates:
Things keep breaking
I don't know why

Wayland works now that I set some Nvidia kernal setting

Audio seems to have maybe fixed itself (discord was having major issue) but now my audio devices have non descript labels?

KDE and or Wayland and or whatever the hell puts graphics on screen seems to encounter large issues after wake from sleep. This seems less so on Wayland vs when I was using X11, but still happens. I changed some other nvidia settings for resume from wake so maybe that will help. I did make a keyboard shortcut to open Konsole and an alias to restart plasma so at least I have a backup.

Still having mouse cursor issues. FIrefox does NOT respect my cursor theme at all, and the cursor in general is acting weird in Krita now.
